### ISSUE TITLE
Android app (v 1.20) crashes constantly

### ISSUE THREAD

**Issue by rudolfmayer** [mapillary/mapillary_issues#836]
I wanted to take some pictures today, but the app keeps on crashing constantly (at least 10 times, also after restarting the app, and rebooting the phone).

I have version 1.20, and initially suspected the "process images in background" setting to be the potential cause, but even deactivating that doesn't help. I sent one of the android crash logs, maybe that can help.

**Comment by peterneubauer** [comment]
Mmh, didn't get a log yet ...

**Comment by rudolfmayer** [comment]
I will try again to get a bug report and send it....

I tried it Mapillary v 1.20 also on another phone, S3 mini (the other one is an S4), and it was more stable, still crashed twice in about 45 minutes.

**Comment by polarbearing** [comment]
1.20 is probably too old. The symptoms sound like the white-screen-crash I had reported on March-11 while driving with 1.19. @peterneubauer had reported that fixed on March-12, and I was testing the 'current' versions downloaded March-18 and March-21, both identified as 1.21, which did not have this crash anymore.

I investigate, however, why they are now mostly waiting longer than the 2 sec shooting interval...

**Comment by rudolfmayer** [comment]
I tried to debug this issue a bit more, and it seems the cause was in having set "0" as interval between shots; this was previously possible, and now is prevented by the GUI, but I guess it still used the value that was once set. Now that I changed it to "2", I don't have any of these issues anymore.

I can't totally confirm that assumption as I can't set it back to "0", but it is the most likely cause, thus I am closing this.
